lib/debug/thread_client.rb

```diff
@@ -607,12 +607,19 @@ def iter_consts c, names = {}
 
     def get_consts expr = nil, only_self: false, &block
       if expr && !expr.empty?
-        _self = frame_eval(expr)
-        if M_KIND_OF_P.bind_call(_self, Module)
-          iter_consts _self, &block
-          return
+        begin
+          puts "this is hit"
```

Comment:
@bitwise-aiden I think this is and the one in rescue block are leftovers from debugging?